### ISSUE TITLE
SCSS syntax is working

### ISSUE THREAD

**Issue by Seanmclem** [SurenAt93/monaco-react#49]
Please can you describe how to load comparability with things like scss? It doesn't work and is apparently part of the monaco-editor library by default

**Comment by SurenAt93** [comment]
You just need to pass "scss" as a value for "language" prop. It should look like this;

```js
<Editor
  height="90vh"
  language="scss"
  value="li { display: inline-block; }"
/>
```

**Comment by SurenAt93** [comment]
As you can notice `scss` support is a default in monaco despite `jsx`. There is a long discussion about it and technically you can implement it, as you can implement any custom language you want, but to be honest, I never faced a "one-line" solution for JSX, maybe it's gonna be the default support in a future version of monaco.

**Comment by Seanmclem** [comment]
Yeah that's pretty much what I found. Nothing simple, but all doable. But nothing that I could properly implement in this library is version of the editor. I don't suppose you have any advice on that? Either way all good. Thanks for the library and the SCSS answer

**Comment by SurenAt93** [comment]
Version of the editor? Do you mean the version of monaco-editor? Like, for example, the latest one is 0.19.2.

If you are about that version, you can implement that with [configs](https://github.com/SurenAt93/monaco-react#config). By default, it will load monaco sources from CDN, and in this particular version it will load 0.19.0 (check it [here](https://github.com/SurenAt93/monaco-react/blob/master/src/config/index.js#L3)). But, of course it's customizable, you can [set](https://github.com/SurenAt93/monaco-react#config) the source of monaco what you want, CDN, your local files or whatever you want and which version you want.

Let me know if it covers your question, or I got something wrong.

**Comment by Seanmclem** [comment]
Yes, I was referring to this particular abstraction on top of the regular Monaco editor. Many of the Articles and Guides Online that have talked about using jsx are very specific to their exact implementation, and pretty much all of them seem to be missing some of the specifics of how they implemented it. Like including code Snippets with variables they never defined in the snippet and so on. So a bit hard to follow for someone like me it was a little new to Monaco. But thanks
